packages/extension-browser/scripts/bundle-size.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('fs');
+const glob = require('glob');
+const path = require('path');
+const packageJSON = require('../package.json');
+
+glob(`${path.resolve(__dirname, '../dist')}/webhint-*.zip`, (err, files) => {
+    if (err) {
+        throw new Error(`Validating bundle size failed: ${err}`);
+    }
+
+    if (files.length !== 1) {
+        throw new Error(`Expected one bundle, but found ${files.length}`);
+    }
+
+    const filename = files[0];
+
+    fs.stat(filename, (err, stats) => {
+        if (err) {
+            throw new Error(`Reading bundle failed: ${err}`);
+        }
+
+        const sizeKB = Math.round(stats.size / 1000);
+        const limitKB = Math.round(packageJSON.bundleSize / 1000);
+
+        if (sizeKB > limitKB) {
+            throw new Error(`Bundle size of ${sizeKB}KB exceeds limit of ${limitKB}KB defined in package.json for ${filename}.`);
```

Comment:
What about `console.error` and `process.exitCode = 1`?
Otherwise we will get the callstack and such and I think this script is only executed via the CLI.

Reply:
Personally I prefer unhandled exceptions as they prevent subsequent script from running. I've been using them other scripts and it doesn't appear to spew a bunch of extra data to the console. 

Also per https://nodejs.org/api/process.html#process_process_exit_code:
> If it is necessary to terminate the Node.js process due to an error condition, throwing an uncaught error and allowing the process to terminate accordingly is safer than calling process.exit().

That's not recommending against `process.exitCode`, but it is calling out throwing an unhandled error as an acceptable way to exit.

Reply:
We should probably normalize what we do across all the scripts. I believe I've been using `exitCode` in more than one place.